src/js/api/justification.js

```diff
@@ -0,0 +1,55 @@
+if (
+    typeof require !== "undefined" &&
+    (typeof window === "undefined" || typeof __webpack_require__ !== "undefined") // eslint-disable-line camelcase
+) {
+    // eslint-disable-next-line no-redeclare
+    var base = require("../base");
+    // eslint-disable-next-line no-redeclare
+    var ripe = base.ripe;
+}
+
+/**
+ * Gets the justifications list. Can be optionally filtered by context.
+ *
+ * @param {Object} options An object of options to configure the query and
+ * its results, such as:
+ * - 'filters[]' - List of filters that the query will use to, operators such as
+ * ('in', 'not_in', 'like', 'likei', 'llike', 'llikei', 'rlike', 'rlikei', 'contains'),
+ * (eg: 'context:eq:42') would filter by the 'context' that equals to '42'.
+ * - 'sort' - List of arguments to sort the results by and which direction
+ * to sort them in (eg: 'id:ascending') would sort by the id attribute in ascending order,
+ * while (eg: 'id:descending')] would do it in descending order.
+ * - 'skip' - The number of the first record to retrieve from the results.
+ * - 'limit' - The number of results to retrieve.
+ * @param {Function} callback Function with the result of the request.
+ * @returns {XMLHttpRequest} The XMLHttpRequest instance of the API request.
+ */
+ripe.Ripe.prototype.getJustifications = function(context = null, options, callback) {

```

Comment:
Having an optional with a default (context) followed by other parameters without it is a bit weird. Is that the way we're doing for other API methods?